docs/Backends.md

```diff
@@ -214,6 +214,23 @@ The `tools/ClassGen/Backends/CPU/CPUSpecificNodes.h` and
 `tools/ClassGen/Backends/CPU/CPUSpecificInstrs.h` files are included in
 `tools/ClassGen/NodeGen.cpp` and `tools/ClassGen/InstrGen.cpp`, respectively.
 
+#### Backend Parameterized Tests
+
+Glow provides several test suites that are parameterized by backend.  An
+example of such a suite is `tests/unittests/OperatorTest.cpp`, which defines
+small tests of Glow operators.  These tests can be executed against any backend
+to check compliance.
+
+These tests will only be run for a backend if a corresponding
+`lib/Backends/$BACKEND/tests` directory is found and contains a corresponding
+`${BACKEND}${TEST}.cpp` file containing a blacklist definition, e.g.:
+```
+std::set<std::string> glow::backendTestBlacklist = {};
+```
+
+This blacklist can be used to exclude any unsupported tests while a backend is
```

Comment:
Nice! It would be still useful to give an example of a shell-command to quickly blacklist all/some tests.

Reply:
Oops, added that to the internal diff and forgot to push to GH.  Should be here in a minute :).

Reply:
Oops added that to the internal diff only :).  Pushing it github now!